src/ExportableMaterial.cpp

```diff
@@ -172,6 +178,196 @@ void ExportableMaterialPBR::convert(ExportableResources &resources,
     }
 }
 
+void ExportableMaterialPBR::loadAiStandard(
+    ExportableResources &resources, const MFnDependencyNode &shaderNode) {
+
+    //References https://docs.substance3d.com/integrations/arnold-5-for-maya-157352171.html
+
+    MStatus status;
+    const auto shaderObject = shaderNode.object(&status);
+    THROW_ON_FAILURE(status);
+
+    auto &args = resources.arguments();
+    args.assignName(m_glMaterial, shaderNode.name().asChar());
+
+    float opacityFactor = 1.f;
+
+    auto hasOpacity =
+        getScalar(shaderObject, "TransmissionWeight", opacityFactor);
```

Comment:
Buggy?

Reply:
What would be buggy here? I'm not familiar with the Arnold material, so reviewing is a bit tricky without digging into the API of that material myself.

Reply:
I can’t find the correct attribute here for opacity and the documentation is lacking. It also requires the transparency check.

Reply:
Well, in that case, I can't help you right now with this. 

If the AI export works for you, then I'm ready to approve this code, just let me know.